doc/api/cli.md

```diff
@@ -815,6 +815,22 @@ changes:
 Specify the `module` containing exported [module customization hooks][].
 `module` may be any string accepted as an [`import` specifier][].
 
+### `--experimental-minimum-coverage`
```

Comment:
If we keep adding more and more flags, they should probably at least be scoped consistently, e.g., `--experimental-minimal-test-coverage`?